docs/release-notes/index.md

```diff
@@ -71,6 +71,14 @@ CHILI GraFx [Environment API 1.1.9](https://sandbox1.chili-publish-sandbox.onlin
 
 ![releasenotes](https://github.com/chili-publish/grafx-documentation/assets/122599725/b0b9bb66-1f25-4c70-bd95-e4e0c7e94f74)
 
+CHILI GraFx [Platform API](https://api.chiligrafx.com/swagger/index.html)
+
+### Improvements
+
+- GET /subscription/id and /subscription/id/renders were migrated to use GUIDs
+Now all our end points use GUIDs for identifying entities
```

Comment:
Here and in the next line you write 'end point' but everywhere else we write 'endpoint'.